src/hooks/meme/useLaunchMeme.ts

```
@@ -0,0 +1,75 @@
+        return;
+      }
+      if (statusRef.current === AsyncRequestStatus.PENDING) {
+        return;
+      }
+      setStatus(AsyncRequestStatus.PENDING);
+      statusRef.current = AsyncRequestStatus.PENDING;
+      try {
+        let image = info?.image || "";
+        if (!image && info.imageFile) {
+          image = await uploadImage(info.imageFile);
+          delete info.imageFile;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)